src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -137,8 +137,10 @@ public function dump(array $options = []): string|array
             'debug' => true,
             'hot_path_tag' => 'container.hot_path',
             'preload_tags' => ['container.preload', 'container.no_preload'],
-            'inline_factories_parameter' => 'container.dumper.inline_factories',
-            'inline_class_loader_parameter' => 'container.dumper.inline_class_loader',
+            'inline_factories_parameter' => 'container.dumper.inline_factories', // @deprecated
+            'inline_class_loader_parameter' => 'container.dumper.inline_class_loader', // @deprecated
```

Comment:
```suggestion
            'inline_factories_parameter' => 'container.dumper.inline_factories', // @deprecated since Symfony 6.3
            'inline_class_loader_parameter' => 'container.dumper.inline_class_loader', // @deprecated since Symfony 6.3
```

to ease BC layers' removals

Reply:
Fair enough, could we agree on `To be removed in 7.0` instead?

Reply:
Consistency-first I'd say 🤷‍♂️ 
<img width="616" alt="Screenshot 2022-12-05 at 13 27 45" src="https://user-images.githubusercontent.com/7502063/205637391-b8a7ea41-3e3d-42ea-9eae-f138a9b7a479.png">

Reply:
OK 😅, done, thanks!